lib/happo/public/HappoDiffs.jsx

```diff
@@ -3,24 +3,32 @@ const PropTypes = React.PropTypes;
 const imageObjectStructure = {
   description: PropTypes.string.isRequired,
   viewport: PropTypes.string.isRequired,
-  url: PropTypes.string.isRequired,
+  diff: PropTypes.string.isRequired,
+  previous: PropTypes.string,
+  current: PropTypes.string,
 };
 
 function imageSlug(image) {
   return btoa(image.description +  image.viewport);
 }
 
-function renderImage(image) {
+function renderImageHeading(image) {
```

Comment:
Yeah, you're right. 

Ninja-unrelated pingback: If you want to look into adding a build-step to the process, go ahead. I thought this file would remain pretty small, but it's obvious that it's about to grow too big pretty soon.